go_client/getting_started.adoc

```diff
@@ -12,229 +12,29 @@ toc::[]
-can be used to interact with its REST API from the Go programming language.
+{product-title} includes a Client Go (client-go) client library for creating applications to interact with {product-title} and Kubernetes clusters. Using the Go programming language (golang), you write programs to perform actions (such as get, list, watch, create, update, or delete) on {product-title} and Kubernetes resources (such as pods, services, and replicationcontrollers). 
 
-Both the client library and API object definitions are versioned, with separate
-branches corresponding to major versions of {product-title}.  It is recommended
-to match the client library version you use to the version of {product-title}
-with which you are interacting.
+Creating client-go applications provides an efficient way of programatically engaging activities to develop, build, deploy, and run pods in the same ways you might from the oc command line. Using client-go programs, you can adapt your {product-title} clusters to facilitate the CI/CD pipeline that best suits your organization.
 
-It is also recommended to use link:https://glide.sh/[Glide] to manage your Go
-libraries: if you do so, Glide will automatically ensure that {product-title}
-client library and its dependencies will be included in your project.
+For client-go applications, you use a link:https://github.com/openshift/client-go[client library] and link:https://github.com/openshift/api[API object definitions] to create applications that interact with the OpenShift REST API. The openshift/client-go library is designed to be compatible with and complementary to the kubernetes/client-go library. To get started, we recommend you follow these general steps:
 
-The following example connects to {product-title} using a pre-existing
-kubeconfig file, then lists all pods and builds in the current namespace.  Copy
-it as main.go into a new directory under your GOPATH.
+1. **Read the link:https://github.com/kubernetes/client-go/blob/master/README.md[client-go README]**: This file helps you understand the components of Kubernetes client-go, as well as which client-go versions you should use with different versions of Kubernetes. You can also see link:https://github.com/kubernetes/client-go/tree/master/examples/in-cluster-client-configuration[in-cluster] and link:https://github.com/kubernetes/client-go/tree/master/examples/out-of-cluster-client-configuration[out-of-cluster] examples of go client applications. Refer to link:https://github.com/openshift/client-go[OpenShift Client in Go] for specifics about OpenShift compatibility issues.
 
-.$GOPATH/src/gettingstarted/main.go
-[source, go]
-----
-package main
+1. **Read the link:https://github.com/openshift/client-go/blob/master/INSTALL.md[client-go INSTALL]: This helps you set up your client-go environment by vendoring (downloading) the correct code and helps you understand the tools you need to create applications.  Dependency management in particular is addressed.
```

Comment:
Unless vendoring is a common client-go term, s/vendoring (downloading)/downloading
s/helps you understand/explains
There's an extra space after "applications."
I'd s/applications.  Dependency management in particular is addressed./applications and manage dependencies.